src/main/cpp/src/ftos_converter.cuh

```diff
@@ -1424,4 +1475,15 @@ __device__ inline int format_float(double value, int digits, bool is_float, char
   }
 }
 
+//===== json_parser utility =====
+
+__device__ inline int double_normalization(double value, char* output)
```

Comment:
Let's consider renaming this to `normalize_double()`.
Also, consider a function doc line indicating the protocol of returning the number of characters if `output` is `nullptr`.

Reply:
Will do in a follow-on.